spec/factories/registration.rb

```diff
@@ -3,6 +3,7 @@
 FactoryGirl.define do
   factory :registration do
     user
-    conference
+    conference { create(:conference, registration_period: create(:registration_period, start_date: 3.days.ago)) }
```

Comment:
We can limit access to Registration with authorization, which also allows us to be flexible with organizers/admins, whom we may want to allow to create a registration after the deadline is over. Should that be enough without a Registration validation?